packages/components/src/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactElement, ReactNode } from "react";
+import { useAssert } from "@jobber/hooks/useAssert";
+import { ComboboxContent } from "./components/ComboboxContent";
+import { ComboboxAction } from "./components/ComboboxAction";
+import { ComboboxContextProvider } from "./ComboboxProvider";
+import {
+  ComboboxTriggerButton,
+  ComboboxTriggerChip,
+} from "./components/ComboboxTrigger";
+
+export interface ComboboxProps {
```

Comment:
Naive, new person question: We have props here, but a types file as well in this directory. Is there a reason we use both patterns? I'm happy to see a type file, but I would want to see all the types in there (including props) just to make sure we're separating concerns (types in the type file, components in the tsx files). If the pattern though is props with the component, every other type in the type file, that's okay too! :)

Reply:
I have a couple guesses but not an informed answer - I think this would be a good team discussion to bring up at Office Hours or after stand up so that we're all informed 😃

Reply:
to be perfectly honest I think it might have been partly out of fear

at one point we tried to be tidy little typers with our types all defined in once place and we got ourselves into a fun little dependency cycle hell 🙃 

though, that was with a bit of a different setup... now with all the types in a single file, that _shouldn't_ be the case, and it's probably a bit of cleanup we should have done because I agree it would be much nicer to have all the exported types/interfaces colocated

if an interface is only used locally and would never ever need to be used outside, then I'm a little less certain it should be in the types file but I could easily be convinced otherwise on that